recipes-kernel/linux/files/set-cmdline.cfg

```diff
@@ -0,0 +1 @@
+CONFIG_CMDLINE="console=ttyAMA0 earlycon=xen earlyprintk=xen clk_ignore_unused"
```

Comment:
again about CMD line? Is it used? I probably can be overriden with cmdline.txt

Reply:
Kernel adds this to cmd line got from bootloader. A'll remove this